### PR TITLE
[infra] Improve test app UIs and usability, switch to fast pass rendering

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

Improved multiple test pages to be usable without UIA software, also switched to fast pass rendering where reasonable.

For math fun facts, check out the commit messages of my commits (non merge commits).
## Motivation and Context

Make test app more usable and friendly.
## How Has This Been Tested?

Tested manually, also ran selection of interaction tests.

### CODE REVIEW

**Comment by StephenLPeters** [comment]
What is the goal of these changes?

**Comment by chingucoding** [comment]
@StephenLPeters Pretty much the same as #1793. Improve UI, fix pages where not everything could be accessed without UIA on CI dimensions or the pages where hard to use due to difficult contrast ratios. As part of this, I also went through all the TextBlocks and updated them to use fast pass rendering. Not sure though why the commit history looks the way it does though. Also the latest commit was an accident, removed that now.

**Comment by chingucoding** [comment]
Fixed the commit history, sorry that so many folks were pinged by my lacking rebasing skills. This commit history makes a lot more sense.

File: dev/RadioButtons/InteractionTests/RadioButtonsTests.cs [hunk 1]
```diff
@@ -142,7 +142,7 @@ public void FocusComingFromAnotherRepeaterTest()
                 VerifySelectedFocusedIndex(2, true);
 
                 // Move focus to the first RadioButtons control by pressing TAB again
-                KeyboardHelper.PressKey(Key.Tab);
+                KeyboardHelper.PressKey(Key.Tab, ModifierKey.Shift);
```
Comment (ranjeshj) [inline dev/RadioButtons/InteractionTests/RadioButtonsTests.cs:145]: Is this intentional ?

File: dev/RadioButtons/TestUI/RadioButtonsFocusPage.xaml.cs [hunk 2]
```diff
@@ -20,13 +21,7 @@ public RadioButtonsFocusPage()
 
         private void OnLoaded(object sender, RoutedEventArgs e)
         {
-            ChangeTestFrameVisibility(Visibility.Collapsed);
```
Comment (ranjeshj) [inline dev/RadioButtons/TestUI/RadioButtonsFocusPage.xaml.cs:23]: >Collapsed [](start = 49, length = 9)

why was this needed before ?

**Comment by chingucoding** [comment]
All tests should work now, could we merge this PR then or are there other things that need addressing?

**Comment by chingucoding** [comment]
@StephenLPeters A few verification tests failed, but I have no idea why since those aren't actually affected by the test app UI are they? Also if we have verification fails, shouldn't the drop contain the updated files? My guess is that it was just a bad run, maybe the merge with main fixes this.

**Comment by StephenLPeters** [comment]
> @StephenLPeters A few verification tests failed, but I have no idea why since those aren't actually affected by the test app UI are they? Also if we have verification fails, shouldn't the drop contain the updated files? My guess is that it was just a bad run, maybe the merge with main fixes this.

We actually changed how the pipelines are organized a bit which resulted in some changes to the VisualVerification test work flow. The updated documentation is in the developer guide: https://github.com/microsoft/microsoft-ui-xaml/blob/main/docs/developer_guide.md#visual-tree-verification-tests.

Can you post a link to the ADO pipeline and I'll take a look. (or I can dig it up if you don't have it handy.)

**Comment by chingucoding** [comment]
I think I found out why a lot of tests failed: I accidently checked in a change to the test app where it used the Version1 styles instead of the Version2 styles which of course broke a lot of verification tests (and some other tests too).

File: dev/RadialGradientBrush/TestUI/RadialGradientBrushPage.xaml [hunk 3]
```diff
@@ -7,7 +7,7 @@
     xmlns:local="using:MUXControlsTestApp"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:media="using:Microsoft.UI.Xaml.Media"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:contract5Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 5)"
```
Comment (karenbtlai) [inline dev/RadialGradientBrush/TestUI/RadialGradientBrushPage.xaml:10]: nit: new line

File: dev/RadioButtons/InteractionTests/RadioButtonsTests.cs (hunk 1 above)
Comment (karenbtlai) [reply dev/RadioButtons/InteractionTests/RadioButtonsTests.cs:145]: Curious as to why you are removing the bit to hide test app bar?

**Comment by karenbtlai** [comment]
I manually compared the before and after MUXControlsTestApps and they look great! Thanks for doing this.

File: dev/RadioButtons/InteractionTests/RadioButtonsTests.cs (hunk 1 above)
Comment (chingucoding) [reply dev/RadioButtons/InteractionTests/RadioButtonsTests.cs:145]: IIRC, you couldn't really leave the app after that since the only way to get back was using some off screen elements that are triggered through UIA when executing tests.

**Review by karenbtlai — APPROVED** [review]

File: dev/RadialGradientBrush/TestUI/RadialGradientBrushPage.xaml (hunk 3 above)
Comment (chingucoding) [reply dev/RadialGradientBrush/TestUI/RadialGradientBrushPage.xaml]: Good point, fixed now (albeit a nit).

**Comment by chingucoding** [comment]
Thank you for reviewing this large PR @karenbtlai !!!
